src/main/java/exceptions/ExcessiveArgException.java

```diff
@@ -0,0 +1,8 @@
+package exceptions;
+
+/**
+ * Exception class for Steven
+ * Used for when there are excessive arguments for a given command
+ */
+public class ExcessiveArgException extends Exception{
```

Comment:
Can we consider having a space before the brace to better follow the coding standard guidelines?